src/Microsoft.ML.Core/Data/Schema.cs

```diff
@@ -0,0 +1,260 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.ML.Runtime.Data
+{
+#pragma warning disable CS0618 // Type or member is obsolete
+    public sealed class Schema : ISchema
```

Comment:
>Schema [](start = 24, length = 6)

Are you planning on writing actual documentation for this eventually?

Reply:
Will do right now

---
In reply to: [224531123](https://github.com/dotnet/machinelearning/pull/1167#discussion_r224531123) [](ancestors = 224531123)